homeassistant/components/conversation/default_agent.py

```diff
@@ -142,6 +150,18 @@ def async_hass_started(hass: core.HomeAssistant) -> None:
 
     start.async_at_started(hass, async_hass_started)
 
+    async def timer_finished_listener(event: core.Event) -> None:
+        """Forward command from timer to process service."""
+        timer_data = event.data.get("data", {})
```

Comment:
there is a data key in the data key?

Reply:
Removed in favor of hard-coding the Assist command.